winlogbeat/docs/overview.asciidoc

```diff
@@ -1,14 +1,21 @@
 == Overview
 
-Winlogbeat ships Windows event logs to Elasticsearch or Logstash. It installs as
-a Windows service on all versions of Windows since XP.
+Winlogbeat ships Windows event logs to Elasticsearch or Logstash. You can
+install it as a Windows service on Windows XP or later.
```

Comment:
@andrewkroh OK, if you'd like, I can add a note to the installation instructions to mention that Windows XP users will need to install PowerShell.